mbtiles.hpp

```diff
@@ -49,6 +50,8 @@ void mbtiles_write_tile(sqlite3 *outdb, int z, int tx, int ty, const char *data,
 
 void mbtiles_write_metadata(sqlite3 *outdb, const char *outdir, const char *fname, int minzoom, int maxzoom, double minlat, double minlon, double maxlat, double maxlon, double midlat, double midlon, int forcetable, const char *attribution, std::map<std::string, layermap_entry> const &layermap, bool vector, const char *description, bool do_tilestats, std::map<std::string, std::string> const &attribute_descriptions, std::string const &program, std::string const &commandline, std::vector<strategy> const &strategies);
 
+void tilestats(std::map<std::string, layermap_entry> const &layermap1, size_t elements, json_writer &state);
```

Comment:
Exposing this in the header although all the tile stats logic should maybe be refactored out of `mbtiles.hpp` into its own file